### ISSUE TITLE
[MOC] Strategy for handling and tracking PVC resizing

### ISSUE THREAD

**Issue by HumairAK** [operate-first/apps#148]
For MOC cluster we have a backing storage that supports pvc expansion. 
Currently when a user's pvc runs out of space, we manually increase the pvc size, and this doesn't get tracked anywhere. 

It seems to me that we could be storing these manauly adjusted pvcs in GIT, and then whenever someone needs more space they can make a PR with a PVC that ups their allotment, have docs + issue template that we can link whenever we get these sorts of requests. 

WDYT?

**Comment by HumairAK** [comment]
Any suggestions on where these should go? These are just pvcs, so I'd imagine we could just keep them in the kustomization for JH [here](https://github.com/operate-first/apps/tree/master/odh/overlays/moc/jupyterhub)https://github.com/operate-first/apps/issues/148

This is a primarily MOC specific thing, there's no guarantee PVC expansion would exist on another cluster.

**Comment by tumido** [comment]
I'd say have the issue template in the support repo, and the PVCs together with the JupyterHub - this targets the JupyterHub primarily + other apps may require different PVC management (may not be user specific etc)

**Comment by tumido** [comment]
Also, they are specific to a user, which implies specific to a cluster anyways. :+1:

**Comment by 4n4nd** [comment]
I agree we should keep pvc manifests in overlays > moc > jupyterhub > pvcs and an issue template would be neat :+1:

**Comment by 4n4nd** [comment]
would the username/email being in the manifest be an issue?

**Comment by tumido** [comment]
We should encrypt it (but we can do that easily).
